package/boot/uboot-envtools/files/qualcommax_ipq60xx

```diff
@@ -23,6 +23,7 @@ linksys,mr7350)
 	ubootenv_add_mtd "u_env" "0x0" "0x40000" "0x20000"
 	;;
 netgear,wax214|\
+tplink,eap620hd-v2|\
```

Comment:
I think that the convention we're moving towards is to have a dash before the `hd` or `outdoor` . In this case `eap620-hd-v2`. The dash stands  in for the spaces in the product name.